config/environments/development.rb

```diff
@@ -37,9 +37,6 @@
     # Expands the lines which load the assets
     config.assets.debug = true
   end
-
-  # Raise exception on mass assignment protection for Active Record models
-  config.active_record.mass_assignment_sanitizer = :strict
```

Comment:
@peggles2 , let's retain the behavior of raising an error in test/dev environments (which I find more useful than the default of logging the params):
`config.action_controller.action_on_unpermitted_parameters = :raise`

Reply:
`:log` is the default in dev/test environments. I'm only suggesting we change it in dev/test environments, because it makes bug hunting faster. Raising an error was our setting until we upgraded to Rails 4 (I didn't realize the behavior had changed until I tested this story).

Reply:
Also, if we set it to `:raise` and our tests break, then we should address those tests and/or code.